### PR TITLE
Team Member Page UI

### PR DESCRIPTION
**Pull request by yasmeen45**
## Description

Changed background colour for team member page and hid "add member" button if user is not logged in.
## Updates

Changed background colour in team member page css and added conditional render for "add member" button.

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by yasmeen45** [comment]
[team member page ui, hide add member button if not log in](https://trello.com/c/shyqvIkB/166-team-member-page-ui-hide-add-member-button-if-not-log-in)

File: src/pages/Project/ProjectTeam.tsx [hunk 1]
```diff
@@ -44,34 +49,18 @@ const ProjectTeam: React.FC<ProjectProps> = (props) => {
...
                 <hr />
+
+                {isLoggedIn ?
                 <div className='add-mem-button-div' onClick={() => setViewState(!viewState)}>
                     <div>
                         + Add Member
                     </div>
                 </div>
-                {/* <div>
-                    <Person
```
Comment (ryanyae) [inline src/pages/Project/ProjectTeam.tsx:53]: I think you can just take all of these comments out tbh.

File: src/pages/Project/ProjectTeam.tsx [hunk 2]
```diff
@@ -44,34 +49,18 @@ const ProjectTeam: React.FC<ProjectProps> = (props) => {
...
-                        linkedIn='linkedin.com'
-                        isCurrentMember={false}
-                    />
-                </div> */}
+                :
+                <div></div>}
+                
+
             </div>
-            {/* <div className="team-header">Team Members</div>
```
Comment (ryanyae) [inline src/pages/Project/ProjectTeam.tsx:73]: You can this out as well.

**Review by ryanyae — CHANGES_REQUESTED** [review]
Everything looks good to me. But I was wondering if you were able to fix this gap at the bottom of the page? 
image

**Comment by yasmeen45** [comment]
Done! Gap was fixed and commented out code was already deleted I believe

**Comment by ryanyae** [comment]
Ah yes you are completely right, sorry about that

**Review by ryanyae — APPROVED** [review]

**Review by ryanyae — APPROVED** [review]

**Pull request merged**